exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/FunctionImplementationRegistry.java

```diff
@@ -291,6 +293,10 @@ public boolean isFunctionComplexOutput(String name) {
     return false;
   }
 
+  public LocalFunctionRegistry getLocalFunctionRegistry() {
```

Comment:
Why you need access to local function registry here?

Reply:
For DRILL-6084 , the `WebServer`  needs access to the registry to load the list of functions for rendering in JavaScript.